docs/configuration/index.md

```diff
@@ -1871,6 +1871,11 @@ Note the following guardrails that can be set by the cluster admin to limit the
 
 Note that limiting the subquery by bytes is a newer feature therefore it is experimental as it materializes the results differently.
 
+'maxSubqueryBytes' can be configured to following values:
+1. 'disabled' - It is the default setting out of the box. It disables the subquery's from the byte based limit, and effectively disables this feature.
+2. 'auto' - Druid automatically decides the optimal byte based limit based upon the heap space available and the max number of concurrent queries.
+3. A postive long value - User can manually specify the number of bytes that the results of the subquerys of a single query can occupy on the heap.
+
```

Comment:
Please mention the side effects as well that in terms of when the performance starts degrading.